packages/react-core/src/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -269,6 +269,50 @@ import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-i
 </DescriptionList>;
 ```
 
+### Horizontal with custom termWidth
+
+```js
+import React from 'react';
+import {
+  Button,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListText,
+  DescriptionListGroup,
+  DescriptionListDescription
+} from '@patternfly/react-core';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+<DescriptionList isHorizontal isInlineGrid termWidth="20ch">
```

Comment:
Is the `isInlineGrid` prop necessary for this example? If so, perhaps you should note that in the example and explain why.